src/services/safe-messages/safeMsgSender.ts

```diff
@@ -97,3 +97,30 @@ export const dispatchSafeMsgConfirmation = async ({
     requestId,
   })
 }
+
+const isMessageFullySigned = (message: SafeMessage): message is SafeMessage & { preparedSignature: string } => {
+  return message.confirmationsSubmitted >= message.confirmationsRequired && !!message.preparedSignature
+}
+
+export const dispatchPreparedSignature = async (
```

Comment:
Why don't we pass in the `message` as it is already present in the modal?

Reply:
Nvm. We want to load the message if we were the last signer to avoid waiting for the polling